src/styles/styles.js

```diff
@@ -716,13 +716,11 @@ const styles = {
     textInputDesktop: addOutlineWidth({}, 0),
 
     secureInputEyeButton: {
+        borderTopRightRadius: 6,
+        borderBottomRightRadius: 6,
         paddingHorizontal: 11,
         justifyContent: 'center',
-    },
-
-    secureInput: {
```

Comment:
Since this was removed, we should also remove it's usage [here](https://github.com/Expensify/App/blob/4f50bf1f9ad3183737b8165f150ca16816b54823/src/components/TextInput/BaseTextInput.js#L265).